doc/design/fully_static_graph.md

```diff
@@ -0,0 +1,122 @@
+# Design Doc: Fully Static Graph
+
+## Abstract
+
+We propose the *fully static graph* rule: training and inference must
```

Comment:
What is the relationship between "fully specified nets" and "static nets"?

Reply:
Thanks! I have modified the design doc, this confusion no longer exists.